muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/workspace/workspaceMaterials/materials/index.tsx

```diff
@@ -773,12 +805,14 @@ class WorkspaceMaterials extends React.Component<
 function mapStateToProps(state: StateType) {
   return {
     i18n: state.i18n,
+    status: state.status,
     workspace: state.workspaces.currentWorkspace,
     materials: state.workspaces.currentMaterials,
     materialReplies: state.workspaces.currentMaterialsReplies,
     activeNodeId: state.workspaces.currentMaterialsActiveNodeId,
     workspaceEditMode: state.workspaces.editMode,
     isLoggedIn: state.status.loggedIn,
+    isStudent: state.status.loggedIn && state.status.isStudent,
```

Comment:
You need the loggedIn here?

Also, you have the same infromations 2 times in the props and by doing that, you save writing "this.props.status.isStudent" instead of "this.props.isStudent". The difference is not huge, but you end up mapping more stuff to props. 

I could not find a single case about what's the best practice here, so I guess this is fine. Just wondering if having the same properties many times in the props is actually a good practice.